cg/apps/housekeeper/hk.py

```diff
@@ -243,6 +243,21 @@ def last_version(self, bundle: str) -> Version:
             .first()
         )
 
+    def get_version_by_version_id(self, version_id: int) -> Optional[Version]:
+        """Get the version corresponding to the given id in Housekeeper."""
+        return self._store.get_version_by_id(version_id)
+
+    def get_all_non_archived_spring_files(self) -> List[File]:
+        """Return all files which are not marked as archived in Housekeeper."""
+        return (
+            self._store._get_query(table=File)
+            .join(Archive)
+            .filter(File.id not in Archive)
+            .filter(SequencingFileTag.SPRING in File.tags)
+            .filter()
+            .all()
+        )
```

Comment:
For consistency this should move into the housekeeper repo, and follow the structure of filter_functions there